src/components/Person.tsx

```diff
@@ -0,0 +1,61 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person as PersonType } from '../types';
+
+type Props = {
+  person: PersonType;
+  people: PersonType[];
+};
+
+export const Person: React.FC<Props> = ({ person, people }) => {
+  const { slugParam } = useParams();
+  const { name, sex, born, died, slug, motherName, fatherName } = person;
+
+  const findParent = (parentName: string | null) => {
+    if (!parentName) {
+      return '-';
+    }
+
+    const parent = people.find(man => man.name === parentName);
+
+    return parent ? (
+      <Link
+        to={`/people/${parent.slug}`}
+        className={cn('', {
+          'has-text-danger': parent.sex === 'f',
+        })}
+      >
+        {parent.name}
+      </Link>
+    ) : (
+      parentName
+    );
+  };
+
+  const mother = findParent(motherName);
+  const father = findParent(fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      className={person.slug === slugParam ? 'has-background-warning' : ''}
```

Comment:
u always used the library - `classNames`. Use it here too.
Never mix ur code styles